qutebrowser/browser/commands.py

```diff
@@ -254,8 +261,10 @@ def openurl(self, url=None, implicit=False,
             if not window and i > 0:
                 tab = False
                 bg = True
-            if tab or bg or window:
-                self._open(cur_url, tab, bg, window, not implicit)
+
+            if tab or bg or window or private:
+                self._open(cur_url, tab, bg, window, explicit=not implicit,
+                           private=private)
```

Comment:
Note that means private browsing isn't inherited when not using `--private`, i.e. `:open -w` in a private window will open a non-private one - I think that's what one would expect?